orchestra/tests/test_orchestra_api.py

```diff
@@ -0,0 +1,168 @@
+import json
+from unittest.mock import patch
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from orchestra.models import Todo
+from orchestra.todos.serializers import BulkTodoSerializer
+from orchestra.tests.helpers.fixtures import TodoFactory
+from orchestra.tests.helpers.fixtures import StepFactory
+from orchestra.tests.helpers.fixtures import ProjectFactory
+from orchestra.project_api.auth import SignedUser
+from orchestra.orchestra_api import create_todos
+from orchestra.orchestra_api import get_todos
+from orchestra.orchestra_api import update_todos
+from orchestra.orchestra_api import delete_todos
+
+
+class TodoAPITests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.request_client = APIClient(enforce_csrf_checks=True)
+        self.request_client.force_authenticate(user=SignedUser())
+        self.project = ProjectFactory()
+        self.step = StepFactory()
+
+    def _pause(self):
+        self.assertTrue(False)
+
+    @patch('orchestra.orchestra_api.requests')
+    def test_create_todos(self, mock_request):
+        # This converts `requests.post` into DRF's `APIClient.post`
+        # To make it testable
+        def post(url, *args, **kwargs):
+            kw = kwargs.get('data', '')
+            data = json.loads(kw)
+            return_value = self.request_client.post(url, data, format='json')
+            return_value.text = json.dumps(return_value.data)
+            return return_value
+
+        mock_request.post = post
+        data = [
+            {
+                'title': 'Testing title {}'.format(x),
+                'project': self.project.id,
+                'step': self.step.id
+            } for x in range(10)
+        ]
+        result = create_todos(data)
+        self.assertEqual(len(result), 10)
+        for r in result:
+            self.assertTrue(r['title'].startswith('Testing title'))
```

Comment:
should create calls return all of the data you created, or just IDs? honestly I don't know :)

Reply:
I'd leave it as it is since it's a common practice in Django to return created objects.